extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java

```diff
@@ -10,23 +10,28 @@
  */
 @ConfigGroup
 public class MicroprofileMetricsConfig implements MicrometerConfig.CapabilityEnabled {
+    // @formatter:off
     /**
      * Microprofile Metrics support.
```

Comment:
```suggestion
     * Eclipse MicroProfile Metrics support.
```
While we're at it (we need Eclipse the first time we mention it).